stdeb/util.py

```diff
@@ -879,6 +879,15 @@ def __init__(self,
         else:
             build_deps.append( 'debhelper (>= %s)'%DH_MIN_VERS )
 
+        try:
+            no_default_build_deps = \
+                cfg.get('DEFAULT', 'No-Default-Build-Deps')\
```

Comment:
Instead of accessing the config directly this should use `parse_val()` instead.

Please document the new configuration option in the README.